processing/src/main/java/io/druid/segment/filter/Filters.java

```diff
@@ -157,6 +163,45 @@ public void remove()
     );
   }
 
+  public static ValueMatcher getTimeValueMatcher(
```

Comment:
How about calling this `getLongValueMatcher` so it can be used with other long dims too, when the time comes?